mockResponse.json

```
@@ -0,0 +1,38 @@
+{
+    "name": "Rohini",
+    "email": "roh3290a@gmail.com",
+    "password": "HalogenIsTheBest2021",
+    "new-user": "true",
+    "outer-ring-colors": [
```

You are absolutely on the right track here with grouping properties together with `favorite-ring-order`, but we can actually make this simpler and easier to change in the future by grouping them more together with each other. 

For example, we can add a property called `palettes` that will be an array of objects. Each object will contain info about one single palette, including it's name, ring colors and gradient colors.

```suggestion
    "palettes": [
        {
        // this object would contain properties like "ringName", "ringColor", etc.
        },
        {
        // And this would contain all the info for a different palette!
        }
    ]
    "outer-ring-colors": [
```

And since those are all ordered in an array, you don't need the `favorite-ring-order` property